jwst/resample/resample.py

```diff
@@ -363,17 +363,30 @@ def update_exposure_times(self, output_model):
         """Modify exposure time metadata in-place"""
         total_exposure_time = 0.
         exposure_times = {'start': [], 'end': []}
+        effinttm, duration = 0.0, 0.0
         for exposure in self.input_models.models_grouped:
-            total_exposure_time += exposure[0].meta.exposure.exposure_time
-            exposure_times['start'].append(exposure[0].meta.exposure.start_time)
-            exposure_times['end'].append(exposure[0].meta.exposure.end_time)
+            for n, _ in enumerate(exposure):
```

Comment:
You need to keep the original design in the loop here, which only adds up values from `exposure[0]` in each of the grouped models. Each group contains models from multiple detectors in instruments like NIRCam and NIRSpec. But we don't want the times from multiple detectors included in the sums. So just use the first entry in each group.